src/index.ts

```diff
@@ -1,83 +1,99 @@
+export { default as Markdown } from "./components/modules/Markdown/Markdown";
+export { default as SiteInfoInnerPane } from "./components/modules/SiteInfoInnerPane/SiteInfoInnerPane";
+export { TitleBar } from "./components/modules/Titlebar/TitleBar";
+export {
+	VerticalNav,
+	VerticalNavItem
+} from "./components/modules/VerticalNav/VerticalNav";
+export { WorkspaceSwitcher } from "./components/modules/VerticalNav/components/WorkspaceSwitcher";
+export { VirtualList } from "./components/modules/VirtualList/VirtualList";
+export { VirtualTable } from "./components/modules/VirtualTable/VirtualTable";
+export { default as WindowsToolbar } from "./components/modules/WindowsToolbar/WindowsToolbar";
 // Overlays
-export {default as FlyModal} from './components/overlays/FlyModal/FlyModal';
-export {default as FlyTooltip} from './components/overlays/FlyTooltip/FlyTooltip';
-export {default as Popup} from './components/overlays/Popup/Popup';
-export {Tooltip} from './components/overlays/Tooltip/Tooltip';
+export { default as FlyModal } from "./components/overlays/FlyModal/FlyModal";
+export { default as FlyTooltip } from "./components/overlays/FlyTooltip/FlyTooltip";
+export { default as Popup } from "./components/overlays/Popup/Popup";
+export { Tooltip } from "./components/overlays/Tooltip/Tooltip";
 // Tables
-export {TableList as TableList, TableListRow as TableListRow} from './components/tables/TableList/TableList';
-export {default as TableListRepeater} from './components/tables/TableListRepeater/TableListRepeater';
+export {
+	TableList,
+	TableListRow
+} from "./components/tables/TableList/TableList";
+export { default as TableListRepeater } from "./components/tables/TableListRepeater/TableListRepeater";
+export { default as TableListMultiDisplay } from "./components/tables/TableListMultiDisplay/TableListMultiDisplay";
 // Text
-export {default as List} from './components/text/List/List';
-export {Text} from './components/text/Text/Text';
-export {TextBase} from './components/text/_private/TextBase/TextBase';
-export {Title} from './components/text/Title/Title';
-export {default as Truncate} from './components/text/Truncate/Truncate';
-export {default as IReactComponentProps} from './common/structures/IReactComponentProps';
+export { default as List } from "./components/text/List/List";
+export { Text } from "./components/text/Text/Text";
+export { TextBase } from "./components/text/_private/TextBase/TextBase";
+export { Title } from "./components/text/Title/Title";
+export { default as Truncate } from "./components/text/Truncate/Truncate";
+export { default as IReactComponentProps } from "./common/structures/IReactComponentProps";
 export {
 	IVirtualTableCellRendererDataArgs,
 	IVirtualTableOnChangeRowDataArgs,
 	IVirtualTableRowRendererDataArgs,
 	VirtualTableCellRenderer,
-	VirtualTableRowRenderer,
-} from './components/modules/VirtualTable/VirtualTable';
+	VirtualTableRowRenderer
+} from "./components/modules/VirtualTable/VirtualTable";
```

Comment:
Same thing here, would mind changing these to single quotes? Not sure how it got all funky 🤔